daisy_workflows/build-publish/windows_media/windows-install-media.wf.json

```diff
@@ -0,0 +1,91 @@
+{
+  "Name": "windows-install-media",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-b",
+  "GCSPath": "gs://gce-image-build-bucket/daisy/${USERNAME}",
+  "Vars": {
+    "media_disk": "media-disk",
+    "build_date": "${TIMESTAMP}",
+    "image_prefix": "windows-install-media",
+    "gcs_path": {"Value": "gs://gce-image-archive/windows-install-media", "Description": "GCS path to export image to"},
+    "workflow_root": {"Value": "/workflows", "Description": "Root of github workflows, defaults to /workflows in the container."}
+  },
+  "Sources": {
+    "prepare_install_media.ps1": "./prepare_install_media.ps1",
+    "components": "./components/",
+    "iso/Win_Server_STD_CORE_2022_64Bit_English.iso": "gs://gce-image-build-resources/windows/iso/SW_DVD9_Win_Server_STD_CORE_2022__64Bit_English_DC_STD_MLF_X22-74290.ISO",
```

Comment:
These should probably be vars like we have for the windows image builds
https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy_workflows/build-publish/windows/windows-server-2019-dc-uefi.wf.json

    "media": {
      "Required": true,
      "Description": "GCS or local path to Windows ISO."
    },

Reply:
done